async_rediscache/session.py

```diff
@@ -14,7 +15,7 @@ class RedisSessionNotInitialized(RuntimeError):
 
 
 class RedisSessionNotConnected(RuntimeError):
-    """Raised when trying to access the Redis client before a connection has been created."""
+    """Raised when trying to access the Redis client before connect has been called."""
```

Comment:
```suggestion
    """Raised when trying to access the Redis client before `connect` has been called."""
```
This seems to be the style in the other docstrings